components/config/prefs.rs

```diff
@@ -265,7 +265,7 @@ impl Preferences {
             dom_serviceworker_timeout_seconds: 60,
             dom_servo_helpers_enabled: false,
             dom_servoparser_async_html_tokenizer_enabled: false,
-            dom_shadowdom_enabled: false,
```

Comment:
Can we remove the option entirely. I don't think it makes sense to allow disabling something that is good enough to be on by default.

Reply:
I believe we usually keep the preference around for a few months to see if things break. 

For example:
* https://github.com/servo/servo/pull/34294
* https://github.com/servo/servo/pull/33186

Reply:
Okay. In the case of flexbox in that moment there was still a lot of bugs to fix. I'm okay with keeping it around for a little while and removing it later though.